iLVu2Client/src/Componets/App.jsx

```diff
@@ -3,11 +3,19 @@ import LoginPage from "../Pages/LoginPage"
 import Registration from "../Pages/RegistrationPage"
 import HomePage from "../Pages/HomePage"
 import { BsChatLeftHeart } from 'react-icons/bs';
+import TrackerPage from "../Pages/TrackerPage";
+import axios from "axios";
 
 
 
 export default function App() {
+
+  function handleLogOut(){
+    axios.get("http://localhost:8000/user/user_logout").then(()=>{alert("user has Logged out")})
```

Comment:
Are you just assuming this will be successful what happens if it's unsuccessful do you still want this behavior?